plugins/interactive.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: interactive
+spec:
+  version: v1.0.3
+  homepage: https://github.com/similarweb/kubectl-interactive
+  shortDescription: A kubectl plugin that provides interactive, easy-to-use and flexible interface
```

Comment:
this doesn't explain nearly anything :) the phrase. has the same level of detail as the word "interactive".

Reply:
changed, hope it makes sense now